docs/project/contribution_tools.md

```diff
@@ -53,9 +53,8 @@ typical tool setup flow is:
 2.  Install [main tools](#main-tools) and any desired
     [optional tools](#optional-tools).
 3.  Set up the [git](https://git-scm.com/) repository:
-    -   In GitHub, create a fork for development at
-        https://github.com/carbon-language/carbon-lang.
-    -   `gh repo clone USER/carbon-lang`, or otherwise clone the fork.
+    -   `gh repo fork --clone carbon-language/carbon-lang`: this will both
+         create a GitHub fork and clone the repository locally
```

Comment:
```suggestion
    -   `gh repo fork --clone carbon-language/carbon-lang`: this will both
        create a GitHub fork and clone the repository locally
```